docs/reference/builder.md

```diff
@@ -848,7 +848,25 @@ named `arr[0].txt`, use the following;
     ADD arr[[]0].txt /mydir/    # copy a file named "arr[0].txt" to /mydir/
 
 
-All new files and directories are created with a UID and GID of 0.
+All new files and directories are created with a UID and GID of 0, unless the
+optional `--chown` flag is used to provide a user and group definition which
+will be used to change the ownership of the created files and directories. The
```

Comment:
How about "optional `--chown` flag specifies a given username, groupname, or UID/GID combination."

(if you only want to give a GID, how does Docker know it's not a UID? I think if you only provide a single integer it's always assumed to be a UID. Maybe best to state this too.)